tests/docker_cluster.py

```diff
@@ -419,14 +434,6 @@ def write_content_to_host(self, content, path, host):
     def copy_to_host(self, source_path, dest_host):
         shutil.copy(source_path, self.get_local_mount_dir(dest_host))
 
-    @nottest
-    def clean_up_presto_test_images(self):
```

Comment:
Let's sync on this. It's not clear to me how the place we're calling this currently is accomplishing cleaning up the images when we run this from intellij